.eslintrc.yaml

```diff
@@ -13,6 +13,8 @@ plugins:
   - 'eslint-plugin-react-native'
   - 'eslint-plugin-react-hooks'
   - '@tanstack/eslint-plugin-query'
+  - 'simple-import-sort'
+  - 'import'
```

Comment:
I know eslint supports omitting the "eslint-plugin-" prefix, but I prefer to keep it… makes it a lost easier for me to track down which package provides it, especially with scoped packages.

Reply:
Oh hmm I thought I was looking at the docs for that one, I'll take a look again.

Reply:
I'm in the minority, for sure. 

If you have a plugin called "eslint-plugin-import", you can put either the full name "eslint-plugin-import" or you can omit the "eslint-plugin-" prefix and just put "import". 

For scoped packages… I don't know. "ts-eslint/eslint-plugin" becomes … just "\@ts-eslint"? (See, this is why I use the full form lol)